src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -9,43 +9,59 @@ docs_area: manage
 {% include_cached feature-phases/limited-access.md %}
 {{site.data.alerts.end}}
 
-[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) autoprovisioning centralizes and automates provisioning and deprovisioning of {{ site.data.products.db }} organization users and groups from your identity provider (IdP).
+[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) helps centralize and automate provisioning, deprovisioning, and identity management for {{ site.data.products.db }} organization users and groups from your identity provider (IdP).
 
 Rather than using [invitations](managing-access.html#invite-team-members-to-an-organization) or self-service [autoprovisioning](cloud-org-sso.html#autoprovisioning), SCIM autoprovisioning tasks are performed centrally by a team of IAM admins in your IdP, who manage the assignment of your organization's users to your organization's app integrations. This page describes SCIM provisioning and shows how to configure and use it to provision {{ site.data.products.db }} users.
```

Comment:
```suggestion
Rather than using [invitations](managing-access.html#invite-team-members-to-an-organization) or self-service [autoprovisioning](cloud-org-sso.html#autoprovisioning), SCIM autoprovisioning tasks are performed centrally by a team of identity and access management (IAM) admins in your IdP, who manage the assignment of your organization's users to your organization's app integrations. This page describes SCIM provisioning and shows how to configure and use it to manage {{ site.data.products.db }} users.
```